### PR TITLE
Test seeding configuration to EdgeX in UC

### PR DESCRIPTION
**Pull request by MonicaisHer**

### CODE REVIEW

**Comment by farshidtz** [comment]
You may want to use latest/stable snaps to have a working and stable environment.

File: test/suites/ubuntu-core-edgex-image/test.sh [hunk 1]
```diff
@@ -10,3 +10,14 @@ do
...
+ssh "$USER@localhost" -p 8022 "snap get edgexfoundry security -d; printf '\n'"
+
+# check the status of the device-virtual service within the emulator
+ssh "$USER@localhost" -p 8022 "snap services edgex-device-virtual; printf '\n'"
+
+# verify that device-virtual has the startup message set from the gadget within the emulator
+ssh "$USER@localhost" -p 8022 'snap logs -n=all edgex-device-virtual | grep "Startup message"; printf '\n''
+
+# access the service endpoints via API Gateway outside of the emulator
+curl --insecure --show-err https://localhost:8443/core-data/api/v2/ping
```
Comment (MonicaisHer) [inline test/suites/ubuntu-core-edgex-image/test.sh:23]:
```
$ curl --insecure --show-err https://localhost:8443/core-data/api/v2/ping
{"message":"Unauthorized"}
```

File: test/suites/ubuntu-core-edgex-image/test.sh [hunk 2]
```diff
@@ -10,3 +10,12 @@ do
...
 
+# verify that the security is avaliable as a snap option of edgexfoundry within the emulator
+ssh "$USER@localhost" -p 8022 "snap get edgexfoundry security -d; printf '\n'"
+
+# check the status of the device-virtual service within the emulator
+ssh "$USER@localhost" -p 8022 "snap services edgex-device-virtual; printf '\n'"
+
+# verify that device-virtual has the startup message set from the gadget within the emulator
+ssh "$USER@localhost" -p 8022 'snap logs -n=all edgex-device-virtual | grep "Startup message"; printf '\n''
+
```
[the remark this answers is not in the document]
Comment (MonicaisHer) [reply test/suites/ubuntu-core-edgex-image/test.sh:21]:
```
$ curl --insecure --show-err https://localhost:59880/api/v2/ping
curl: (35) OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to localhost:59880 
```

[the remark this answers is not in the document]
Comment (farshidtz) [reply test/suites/ubuntu-core-edgex-image/test.sh:21]: There is no TLS termination by services; the URL scheme should be `http`. The reverse proxy does TLS term on port 8443.

**Review by farshidtz — APPROVED** [review]
Looks good. I updated the title.

File: test/suites/ubuntu-core-edgex-image/test.sh (hunk 1 above)
Comment (farshidtz) [reply test/suites/ubuntu-core-edgex-image/test.sh:23]: I'm surprised that this request even works. With security disabled, the reverse proxy should not be running.

**Pull request merged**